modules/jwt_auth_refresh/src/JwtRefreshTokensInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Drupal\jwt_auth_refresh;
+
+use Drupal\Core\Session\AccountInterface;
+
+interface JwtRefreshTokensInterface {
```

Comment:
Now I get the singular/plural difference. I haven't thought about this long enough to have a strong opinion. Did you consider extending ContentEntityStorageBase for this?